zuul.d/project-templates.yaml

```diff
@@ -326,151 +326,97 @@
 - project-template:
     name: ansible-collections-cisco-iosxr
     check:
-      jobs:
+      jobs: &ansible-collections-cisco-iosxr-jobs
```

Comment:
lets sort this job order now, should help make reviewing easier for any missing jobs.